### DISCUSSION TITLE
Watch folder and QSV - more info from testing below

### DISCUSSION THREAD

**Discussion by johnvick** [jlesage/docker-handbrake#313] [General]
I have previously posted on this topic:

https://github.com/jlesage/docker-handbrake/issues/210
https://github.com/jlesage/docker-handbrake/discussions/308

But never found a solution. 

The problem is that the watch folder does not always work with a QSV 265 profile. It works reliably with CPU encoding profiles and if files are manually loaded via the web interface, when QSV works. 

Having tested on three devices with lots of mixed 264 and 265 encoded files I have observed that in almost every case the source files that don’t work with the watch folder are 264 encoded. 

To test this further I created two 5 minute videos, one 264 and the other 265 encoded.

In all cases the 264 encoded file failed in the watch folder – 0 byte file in the output folder.
The 265 encoded files worked OK.

The devices are:

Device 1: Intel(R) Pentium(R) CPU J4205 @ 1.50GHz (10 bit QSV 265 n/a) (Proxmox LXC)
Device 2: Intel(R) Pentium(R) Gold G5400 CPU @ 3.70GHz (Debian OMV 6)
Device 3: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (Proxmox LXC)

Lowpower=0 in all cases.

Here is the log from a failed encode on Device 3:

________________________________________________________________________________

------- CONVERSION OUTPUT Thu Jan  4 12:12:56 NZDT 2024 -------
[12:12:56] Compile-time hardening features are enabled
Cannot load libnvidia-encode.so.1
libva info: VA-API version 1.20.0
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
libva info: VA-API version 1.20.0
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
[12:12:56] hb_display_init: attempting VA driver 'iHD'
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
[12:12:56] hb_display_init: using VA driver 'iHD'
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
[12:12:56] qsv: is available on this system
[12:12:56] hb_init: starting libhb thread
[12:12:56] thread 7fe1264d0b38 started ("libhb")
HandBrake 1.7.2 (2023122500) - Linux x86_64 - https://handbrake.fr
6 CPUs detected
Opening /watch/Example 264-001.mkv...
[12:12:56] CPU: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
[12:12:56]  - Intel microarchitecture Tiger Lake
[12:12:56]  - logical processor count: 6
[12:12:56] Intel Quick Sync Video support: yes
[12:12:56] Intel Quick Sync Video integrated adapter with index 0 and renderD128
[12:12:56] Impl mfx-gen library path: /usr/lib/libmfx-gen.so.1.2.9
[12:12:56]  - Intel Media SDK hardware: API 2.9 (minimum: 1.3)
[12:12:56]  - Decode support:  h264 hevc (8bit: yes, 10bit: yes) av1 (8bit: yes, 10bit: yes)
[12:12:56]  - H.264 encoder: yes
[12:12:56]     - preferred implementation: hardware (any) via ANY
[12:12:56]     - capabilities (hardware):  lowpower breftype icq+la+i+downs vsinfo chromalocinfo opt1 opt2+mbbrc+extbrc+trellis+repeatpps+ib_adapt+nmpslice
[12:12:56]  - H.265 encoder: yes (8bit: yes, 10bit: yes)
[12:12:56]     - preferred implementation: hardware (any) via ANY
[12:12:56]     - capabilities (hardware):  lowpower bpyramid icq vsinfo masteringinfo cllinfo opt1
[12:12:56]  - AV1 encoder: no
[12:12:56] hb_scan: path=/watch/Example 264-001.mkv, title_index=1
[12:12:56] thread 7fe12646db38 started ("scan")
Input #0, matroska,webm, from '/watch/Example 264-001.mkv':
  Metadata:
    encoder         : libebml v1.4.5 + libmatroska v1.7.1
    creation_time   : 2024-01-03T23:10:45.000000Z
    MOVIE/ENCODER   : Lavf58.29.100
  Duration: 00:05:00.10, start: 0.000000, bitrate: 2250 kb/s
  Stream #0:0: Video: h264 (High), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 25 fps, 25 tbr, 1k tbn (default)
    Metadata:
      BPS             : 2120881
      DURATION        : 00:05:00.000000000
      NUMBER_OF_FRAMES: 7500
      NUMBER_OF_BYTES : 79533052
      _STATISTICS_WRITING_APP: mkvmerge v82.0 ('I'm The President') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2024-01-03 23:10:45
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:1: Audio: aac (LC), 48000 Hz, stereo, fltp (default)
    Metadata:
      BPS             : 128019
      DURATION        : 00:05:00.096000000
      NUMBER_OF_FRAMES: 14067
      NUMBER_OF_BYTES : 4802283
      _STATISTICS_WRITING_APP: mkvmerge v82.0 ('I'm The President') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2024-01-03 23:10:45
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[12:12:56] Found ffmpeg 0 chapters, container=matroska,webm
[12:12:56] scan: decoding previews for title 1
[12:12:56] scan: preview 1
[12:12:56] fifo_close: trashing 0 buffer(s)
[12:12:56] scan: audio 0x1: aac, rate=48000Hz, bitrate=1 Unknown (AAC LC, 2.0 ch)
[12:12:56] scan: preview 2
[12:12:56] scan: preview 3
[12:12:56] scan: preview 4
[12:12:56] scan: preview 5
[12:12:56] scan: preview 6
[12:12:56] scan: preview 7
[12:12:56] scan: preview 8
[12:12:56] scan: preview 9
[12:12:56] scan: preview 10

Scanning title 1 of 1, preview 10, 100.00 %[12:12:56] using container PAR 1:1
[12:12:56] crop: [0] 58/58/0/0
[12:12:56] crop: [1] 58/58/0/0
[12:12:56] crop: [2] 58/58/0/0
[12:12:56] crop: [3] 58/58/0/0
[12:12:56] crop: [4] 58/58/0/0
[12:12:56] crop: [5] 58/58/0/0
[12:12:56] crop: [6] 58/58/0/0
[12:12:56] crop: [7] 58/58/0/0
[12:12:56] crop: [8] 59/59/0/0
[12:12:56] crop: [9] 59/59/0/0
[12:12:56] crop: less_than_median_frame_count: 0,
[12:12:56] scan: 10 previews, 1920x1080, 25.000 fps, autocrop = 58/58/0/0, aspect 16:9, PAR 1:1, color profile: 1-1-1, chroma location: left
[12:12:56] scan: supported video decoders: avcodec qsv
[12:12:56] Freed 1 buffers of size 1024
[12:12:56] Freed 1 buffers of size 2048
[12:12:56] Freed 1 buffers of size 4096
[12:12:56] Freed 1 buffers of size 8192
[12:12:56] Freed 1 buffers of size 16384
[12:12:56] Freed 1 buffers of size 32768
[12:12:56] Freed 1 buffers of size 65536
[12:12:56] Freed 1 buffers of size 131072
[12:12:56] Allocated 261120 bytes of buffers on this pass and Freed 261120 bytes, 0 bytes leaked
[12:12:56] thread 7fe12646db38 exited ("scan")
[12:12:56] thread 7fe12646db38 joined ("scan")
[12:12:56] libhb: scan thread found 1 valid title(s)
+ Using preset: H.265 QSV 1080p JV
+ title 1:
  + stream: /watch/Example 264-001.mkv
  + duration: 00:05:00
  + size: 1920x1080, pixel aspect: 1/1, display aspect: 1.78, 25.000 fps
  + autocrop: 58/58/0/0
  + chapters:
    + 1: duration 00:05:00
  + audio tracks:
    + 1, Unknown (AAC LC, 2.0 ch) (iso639-2: und)
  + subtitle tracks:
[12:12:57] thread 7fe12646db38 started ("work")
[12:12:57] Starting work at: Thu Jan  4 12:12:57 2024

[12:12:57] 1 job(s) to process
[12:12:57] json job:
{
    "Audio": {
        "AudioList": [],
        "CopyMask": [
            "copy:aac",
            "copy:eac3"
        ],
        "FallbackEncoder": "none"
    },
    "Destination": {
        "AlignAVStart": false,
        "ChapterList": [
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 5,
                    "Seconds": 0,
                    "Ticks": 27008640
                },
                "Name": ""
            }
        ],
        "ChapterMarkers": false,
        "File": "/output/.fC0Sys/Example 264-001.MKV",
        "InlineParameterSets": false,
        "Mux": "mkv",
        "Options": {
            "IpodAtom": false,
            "Optimize": false
        }
    },
    "Filters": {
        "FilterList": [
            {
                "ID": 10,
                "Settings": {
                    "mode": 1
                }
            },
            {
                "ID": 19,
                "Settings": {
                    "crop-bottom": 0,
                    "crop-left": 0,
                    "crop-right": 0,
                    "crop-top": 0,
                    "height": 1080,
                    "width": 1920
                }
            }
        ]
    },
    "Metadata": {},
    "PAR": {
        "Den": 1,
        "Num": 1
    },
    "SequenceID": 0,
    "Source": {
        "Angle": 0,
        "HWDecode": 0,
        "Path": "/watch/Example 264-001.mkv",
        "Range": {
            "End": 1,
            "Start": 1,
            "Type": "chapter"
        },
        "Title": 1
    },
    "Subtitle": {
        "Search": {
            "Burn": true,
            "Default": false,
            "Enable": false,
            "Forced": false
        },
        "SubtitleList": []
    },
    "Video": {
        "Bitrate": 2400,
        "ChromaLocation": 1,
        "ColorInputFormat": 0,
        "ColorMatrix": 1,
        "ColorOutputFormat": 0,
        "ColorPrimaries": 1,
        "ColorRange": 1,
        "ColorTransfer": 1,
        "Encoder": "qsv_h265_10bit",
        "HardwareDecode": 0,
        "Level": "auto",
        "MultiPass": false,
        "Options": "lowpower=0",
        "Preset": "quality",
        "Profile": "auto",
        "QSV": {
            "AdapterIndex": 0,
            "AsyncDepth": 0,
            "Decode": true
        },
        "Tune": "",
        "Turbo": false
    }
}
[12:12:57] qsv: integrated qsv adapter with index 0 has been selected
[12:12:57] Starting Task: Encoding Pass
[12:12:57] work: skipping crop/scale filter
[12:12:57] job configuration:
[12:12:57]  * source
[12:12:57]    + /watch/Example 264-001.mkv
[12:12:57]    + title 1, chapter(s) 1 to 1
[12:12:57]    + container: matroska,webm
[12:12:57]    + data rate: 2250 kbps
[12:12:57]  * destination
[12:12:57]    + /output/.fC0Sys/Example 264-001.MKV
[12:12:57]    + container: Matroska (libavformat)
[12:12:57]  * video track
[12:12:57]    + decoder: h264_qsv 8-bit (yuv420p)
[12:12:57]    + filters
[12:12:57]      + Framerate Shaper (mode=1)
[12:12:57]        + frame rate: 25.000 fps -> constant 25.000 fps
[12:12:57]      + AVFilter (format='pix_fmts=p010le')
[12:12:57]        + format: pix_fmts=p010le
[12:12:57]      + Format (format=p010le)
[12:12:57]    + Output geometry
[12:12:57]      + storage dimensions: 1920 x 1080
[12:12:57]      + pixel aspect ratio: 1 : 1
[12:12:57]      + display dimensions: 1920 x 1080
[12:12:57]    + encoder: H.265 10-bit (Intel Media SDK)
[12:12:57]      + preset:  quality
[12:12:57]      + options: lowpower=0
[12:12:57]      + profile: auto
[12:12:57]      + level:   auto
[12:12:57]      + bitrate: 2400 kbps, pass: 0
[12:12:57]      + color profile: 1-1-1
[12:12:57]      + chroma location: left
[12:12:57] sync: expecting 7502 video frames
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
[12:12:57] hb_display_init: using VA driver 'iHD'
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
[12:12:57] encqsvInit: using encode-only via system memory path
[12:12:57] encqsvInit: H.265/HEVC Main 10 profile @ level 4.0
[12:12:57] encqsvInit: TargetUsage 1 AsyncDepth 6
[12:12:57] encqsvInit: GopRefDist 8 GopPicSize 56 NumRefFrame 4 IdrInterval 0
[12:12:57] encqsvInit: BFramesMax 7 BRefType pyramid
[12:12:57] encqsvInit: RateControlMethod VBR TargetKbps 2400 MaxKbps 2400 BufferSizeInKB 600 InitialDelayInKB 300 BRCParamMultiplier 1
[12:12:57] Writing Metadata to output file...
[12:12:57] thread 7fe125cf0b38 started ("Reader")
[12:12:57] thread 7fe125b13b38 started ("Video decoder (libavcodec)")
[12:12:57] thread 7fe125acfb38 started ("Video Synchronization")
[12:12:57] ffmpeg_read starting chapter 1 at 7200
[12:12:57] thread 7fe125aacb38 started ("Quick Sync Video encoder (Intel Media SDK)")
[12:12:57] thread 7fe125a89b38 started ("Muxer")
[12:12:57] thread 7fe125a66b38 started ("Framerate Shaper")
[12:12:57] thread 7fe125a43b38 started ("AVFilter")
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0
libva info: VA-API version 1.20.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /usr/lib/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_20
libva info: va_openDriver() returns 0

**Comment by TLHD** [comment]
I'm having the exact same problem.
H264 and others as input dont work. Only hevc/265 files do. 
Anyone with a solution for that?

**Reply by TLHD** [reply]
------- CONVERSION OUTPUT Sat Jul 6 11:01:39 UTC 2024 ------- [11:01:39] Compile-time hardening features are enabled Cannot load libnvidia-encode.so.1 libva info: VA-API version 1.21.0 libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 [11:01:39] hb_display_init: attempting VA driver 'iHD' libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 [11:01:39] hb_display_init: using VA driver 'iHD' libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 [11:01:39] qsv: is available on this system [11:01:39] hb_init: starting libhb thread [11:01:39] thread 7f90bf77cb30 started ("libhb") HandBrake 1.8.1 (2024063000) - Linux x86_64 - https://handbrake.fr 6 CPUs detected Opening /watch/testfile1.mkv... [11:01:39] CPU: Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz [11:01:39] - Intel microarchitecture Kaby Lake [11:01:39] - logical processor count: 6 [11:01:39] Intel Quick Sync Video support: yes [11:01:39] Intel Quick Sync Video integrated adapter with index 0 and renderD128 [11:01:39] Impl mfxhw64 library path: /usr/lib/libmfxhw64.so.1.35 [11:01:39] - Intel Media SDK hardware: API 1.35 (minimum: 1.3) [11:01:39] - Decode support: h264 hevc (8bit: yes, 10bit: yes) [11:01:39] - H.264 encoder: yes [11:01:39] - preferred implementation: hardware (any) via ANY [11:01:39] - capabilities (hardware): breftype icq+la+i+downs vsinfo chromalocinfo opt1 opt2+mbbrc+extbrc+trellis+repeatpps+ib_adapt+nmpslice [11:01:39] - H.265 encoder: yes (8bit: yes, 10bit: yes) [11:01:39] - preferred implementation: hardware (any) via ANY [11:01:39] - capabilities (hardware): bpyramid icq vsinfo masteringinfo cllinfo opt1 [11:01:39] - AV1 encoder: no [11:01:39] hb_scan: path=/watch/testfile1.mkv, title_index=1 Input #0, matroska,webm, from '/watch/testfile1.mkv': Metadata: encoder : libebml v1.4.4 + libmatroska v1.7.1 creation_time : 2024-05-14T18:50:20.000000Z Duration: 00:41:49.14, start: 0.000000, bitrate: 1248 kb/s Stream #0:0: Video: h264 (High), yuv420p(tv, bt709, progressive), 720x404, SAR 1:1 DAR 180:101, 23.98 fps, 23.98 tbr, 1k tbn (default) Metadata: BPS : 1030759 DURATION : 00:41:49.132000000 NUMBER_OF_FRAMES: 60159 NUMBER_OF_BYTES : 323288923 _STATISTICS_WRITING_APP: mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit _STATISTICS_WRITING_DATE_UTC: 2024-05-14 18:50:20 _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES Stream #0:1(ger): Audio: aac (LC), 48000 Hz, stereo, fltp (default) Metadata: BPS : 106303 DURATION : 00:41:49.077000000 NUMBER_OF_FRAMES: 117613 NUMBER_OF_BYTES : 33340333 _STATISTICS_WRITING_APP: mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit _STATISTICS_WRITING_DATE_UTC: 2024-05-14 18:50:20 _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES Stream #0:2(eng): Audio: aac (LC), 48000 Hz, stereo, fltp Metadata: BPS : 108468 DURATION : 00:41:49.120000000 NUMBER_OF_FRAMES: 117615 NUMBER_OF_BYTES : 34020152 _STATISTICS_WRITING_APP: mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit _STATISTICS_WRITING_DATE_UTC: 2024-05-14 18:50:20 _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES Stream #0:3(eng): Subtitle: subrip (srt) Metadata: BPS : 76 DURATION : 00:40:50.230000000 NUMBER_OF_FRAMES: 775 NUMBER_OF_BYTES : 23506 _STATISTICS_WRITING_APP: mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit _STATISTICS_WRITING_DATE_UTC: 2024-05-14 18:50:20 _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES Stream #0:4(eng): Subtitle: subrip (srt) Metadata: title : SDH BPS : 78 DURATION : 00:41:43.849000000 NUMBER_OF_FRAMES: 844 NUMBER_OF_BYTES : 24631 _STATISTICS_WRITING_APP: mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit _STATISTICS_WRITING_DATE_UTC: 2024-05-14 18:50:20 _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES [11:01:40] scan: decoding previews for title 1 [11:01:40] scan: audio 0x1: aac, rate=48000Hz, bitrate=1 Deutsch (AAC LC, 2.0 ch) [11:01:40] scan: audio 0x2: aac, rate=48000Hz, bitrate=1 English (AAC LC, 2.0 ch) [11:01:40] scan: 10 previews, 720x404, 23.976 fps, autocrop = 0/0/0/0, aspect 16:9, PAR 1:1, color profile: 1-1-1, chroma location: left [11:01:40] scan: supported video decoders: avcodec qsv [11:01:40] libhb: scan thread found 1 valid title(s) + Using preset: tlh + title 1: + stream: /watch/testfile1.mkv + duration: 00:41:49 + size: 720x404, pixel aspect: 1/1, display aspect: 1.78, 23.976 fps + autocrop: 0/0/0/0 + chapters: + 1: duration 00:41:49 + audio tracks: + 1, Deutsch (AAC LC, 2.0 ch) (iso639-2: deu) + 2, English (AAC LC, 2.0 ch) (iso639-2: eng) + subtitle tracks: + 1, English (UTF-8) + 2, English (UTF-8) [11:01:40] Starting work at: Sat Jul 6 11:01:40 2024 [11:01:40] 1 job(s) to process [11:01:40] json job: { "Audio": { "AudioList": [ { "Bitrate": 0, "DRC": 0.0, "Encoder": "copy:aac", "Mixdown": "none", "PresetEncoder": "copy", "Quality": -3.0, "Samplerate": 0, "Track": 0 }, { "Bitrate": 0, "DRC": 0.0, "Encoder": "copy:aac", "Mixdown": "none", "PresetEncoder": "copy", "Quality": -3.0, "Samplerate": 0, "Track": 1 } ], "CopyMask": [ "copy:mp2", "copy:mp3", "copy:aac", "copy:ac3", "copy:dts", "copy:dtshd", "copy:eac3", "copy:flac", "copy:truehd", "copy:opus" ], "FallbackEncoder": "none" }, "Destination": { "AlignAVStart": false, "ChapterList": [ { "Duration": { "Hours": 0, "Minutes": 41, "Seconds": 49, "Ticks": 225822600 }, "Name": "" } ], "ChapterMarkers": false, "File": "/output/.yGxAnA/testfile1.mkv", "InlineParameterSets": false, "Mux": "mkv", "Options": { "IpodAtom": false, "Optimize": false } }, "Filters": { "FilterList": [ { "ID": 11, "Settings": { "mode": 0 } }, { "ID": 20, "Settings": { "crop-bottom": 0, "crop-left": 0, "crop-right": 0, "crop-top": 0, "height": 404, "width": 720 } } ] }, "Metadata": {}, "PAR": { "Den": 1, "Num": 1 }, "SequenceID": 0, "Source": { "Angle": 0, "HWDecode": 0, "Path": "/watch/testfile1.mkv", "Range": { "End": 1, "Start": 1, "Type": "chapter" }, "Title": 1 }, "Subtitle": { "Search": { "Burn": false, "Default": true, "Enable": true, "Forced": true }, "SubtitleList": [ { "Burn": false, "Default": false, "Forced": false, "Track": 0 }, { "Burn": false, "Default": false, "Forced": false, "Name": "SDH", "Track": 1 } ] }, "Video": { "ChromaLocation": 1, "ColorInputFormat": 0, "ColorMatrix": 1, "ColorOutputFormat": 0, "ColorPrimaries": 1, "ColorRange": 1, "ColorTransfer": 1, "Encoder": "qsv_h265", "HardwareDecode": 0, "Level": "auto", "MultiPass": false, "Options": "", "Preset": "speed", "Profile": "auto", "QSV": { "AdapterIndex": 0, "AsyncDepth": 0, "Decode": true }, "Quality": 27.0, "Tune": "", "Turbo": false } } [11:01:40] qsv: integrated qsv adapter with index 0 has been selected [11:01:40] Skipping subtitle scan. No suitable subtitle tracks. [11:01:40] Starting Task: Encoding Pass [11:01:40] work: skipping crop/scale filter [11:01:40] job configuration: [11:01:40] * source [11:01:40] + /watch/testfile1.mkv [11:01:40] + title 1, chapter(s) 1 to 1 [11:01:40] + container: matroska,webm [11:01:40] + data rate: 1248 kbps [11:01:40] * destination [11:01:40] + /output/.yGxAnA/testfile1.mkv [11:01:40] + container: Matroska (libavformat) [11:01:40] * video track [11:01:40] + decoder: h264_qsv 8-bit (yuv420p) [11:01:40] + filters [11:01:40] + Framerate Shaper (mode=0) [11:01:40] + frame rate: same as source (around 23.976 fps) [11:01:40] + Format (format=nv12) [11:01:40] + Output geometry [11:01:40] + storage dimensions: 720 x 404 [11:01:40] + pixel aspect ratio: 1 : 1 [11:01:40] + display dimensions: 720 x 404 [11:01:40] + encoder: H.265 (Intel Media SDK) [11:01:40] + preset: speed [11:01:40] + profile: auto [11:01:40] + level: auto [11:01:40] + quality: 27.00 (ICQ) [11:01:40] + color profile: 1-1-1 [11:01:40] + color range: tv [11:01:40] + chroma location: left [11:01:40] * subtitle track 1, English (UTF-8) (track 0, id 0x3, Text) -> Passthrough [11:01:40] * subtitle track 2, English (UTF-8) (track 1, id 0x4, Text) -> Passthrough [11:01:40] + name: SDH [11:01:40] * audio track 1 [11:01:40] + decoder: Deutsch (AAC LC, 2.0 ch) (track 1, id 0x1) [11:01:40] + samplerate: 48000 Hz [11:01:40] + AAC Passthru [11:01:40] * audio track 2 [11:01:40] + decoder: English (AAC LC, 2.0 ch) (track 2, id 0x2) [11:01:40] + samplerate: 48000 Hz [11:01:40] + AAC Passthru [11:01:40] sync: expecting 60159 video frames libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 [11:01:40] hb_display_init: using VA driver 'iHD' libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0 [11:01:40] encqsvInit: using encode-only via system memory path [11:01:40] encqsvInit: H.265/HEVC Main profile @ level 3.0 [11:01:40] encqsvInit: TargetUsage 7 AsyncDepth 4 [11:01:40] encqsvInit: GopRefDist 4 GopPicSize 48 NumRefFrame 3 IdrInterval 0 [11:01:40] encqsvInit: BFramesMax 3 BRefType pyramid [11:01:40] encqsvInit: RateControlMethod ICQ ICQQuality 27 libva info: VA-API version 1.21.0 libva info: User environment variable requested driver 'iHD' libva info: Trying to open /usr/lib/dri/iHD_drv_video.so libva info: Found init function __vaDriverInit_1_21 libva info: va_openDriver() returns 0

**Comment by TNFB** [comment]
Hey,
I've found a temporary solution. You can find it here: https://github.com/jlesage/docker-handbrake/issues/347#issuecomment-2222509081
Tell me if it works for you !

**Comment by brandonscript** [comment]
I can't promise that this really is related to this issue or not, but I recently found this happening to me, and it was the use of `{codec}` in the name mask in HandBrake preferences.
